README.md

```diff
@@ -1,42 +1,26 @@
-# FIRST Rust Competition
+<h1 align="center">WPILib-rs</h1>
 
-[![Build Status](https://travis-ci.com/first-rust-competition/first-rust-competition.svg?branch=master)](https://travis-ci.com/first-rust-competition/first-rust-competition)
-[![Crates.io](https://img.shields.io/crates/v/wpilib.svg)](https://crates.io/crates/wpilib/)
-[![Docs.rs](https://docs.rs/wpilib/badge.svg)](https://docs.rs/wpilib)
+<h4 align="center">
+    <a href="https://docs.rs/wpilib/">Documentation</a>
+  | <a href="./CONTRIBUTING.md">Contributing</a>
+</h1>
 
-A monorepo for `wpilib` for programming FRC robots and `cargo-frc` for deploying said code. Currently a pre-alpha WIP.
+<h4 align="center">
+    <img alt="Crates.io" src="https://img.shields.io/crates/v/wpilib?label=wpilib&style=flat-square">
+    <img alt="Crates.io" src="https://img.shields.io/crates/v/wpilib-sys?label=wpilib-sys&style=flat-square">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/bobbbay/wpilib-rs/Continuous%20integration?style=flat-square">
+</h1>
 
-## Getting Started
+<p align="center"> A memory-safe, modular, expressive alternative to programming FRC robots </p>
 
-Parts of this repository are designed to be compiled for a [RoboRIO](http://sine.ni.com/nips/cds/view/p/lang/en/nid/213308), the
-processor used in the FIRST Robotics Competition.
-To cross-compile your code and run Rust on your RoboRIO, follow the instructions in [WPILib's README](wpilib/README.md).
+`WPIlib-rs` is a memory-safe, modular, and expressive strongly-typed low-level alternative to programming FRC robots. This suite of libraries binds and builds upon [WPILib](https://wpilib.org/), enabling [First Robotics Competition](https://www.firstinspires.org/robotics/frc) teams to *safely*, *efficiently*, and *legally* program their robots, with Rust.
 
-Examples can be found in [wpilib-examples](wpilib-examples).
+## Features
 
-To deploy code you write using `wpilib`, use [cargo-frc](cargo-frc).
+ * **Idiomatic abstractions** (on top of WPILib's internal library)
+ * A **low-level interface** (interact directly with the HAL, if needed)
+ * **Directly bound to WPILib** (no fluff in between)
 
-A small project template is available in [quickstart.zip](quickstart.zip).
+## Getting started
 
-## Other Rust Projects
```

Comment:
Is there a reason this section was removed? The links could do with updating to `first-rust-competition` but otherwise the projects are still useful.

Reply:
I removed this because my next PR at the time was going to be adding these to this codebase. Now that I won't be making this PR, I have no problem bringing it back!

Reply:
I don't think `nt-rs` belongs in this monorepo anyway. It's not coupled to anything in here.

Reply:
Sure. Again, I won't be bringing any of those projects into this codebase, so you can rewrite them again :).